app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,880 @@
+    key = "reviews",                 // Attribute key           
+    twoWayKey = "movie",             // Two-way attribute key       
+    onDelete = "cascade"             // On delete action        
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Swift</h3>
+        <div class="ide margin-top-small" data-lang="swift" data-lang-label="Swift SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+    .setProject("5df5acd0d48c2")                // Your project ID
+
+let databases = Databases(client)
+
+databases.createRelationshipAttribute(
+    databaseId: "marvel",           // Database ID              
+    collectionId: "movies",         // Collection ID            
+    relatedCollectionId: "reviews", // Related collection ID    
+    type: "oneToMany",              // Relationship type        
+    twoWay: true,                   // Is two-way               
+    key: "reviews",                 // Attribute key            
+    twoWayKey: "movie",             // Two-way attribute key        
+    onDelete: "cascade"             // On delete action         
+)</code></pre>
+        </div>
+    </li>
+</ul>
+
+<p>The above example adds a relationship between the collections <b>movies</b> and <b>reviews</b>. A relationship attribute with the key <code>reviews</code> is added to the movies collection and another relationship attribute with the key <code>movie</code> is added to the reviews collection.</p>
+
+<p>Relationship attribute keys are used to reference the related collection in queries, so pick something that's intuitive and easy to remember.</p>
+
+<h2><a href="/docs/databases-relations#create-documents" id="create-documents">Creating Documents</a></h2>
+<p>If a collection has relationship attributes, you can create documents in two ways.</p>
+
+<h3><a href="/docs/databases-relations#create-nested" id="create-nested">Created Child Documents</a></h3>
+<p>You can create both the <b>parent</b> and <b>child</b> at once in a relationship by nesting data.</p>
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Web</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>const { Client, Databases, ID } = require('node-appwrite');
+
+const client = new Client()
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2');c              // Your project ID
+
+const databases = new Databases(client);
+
+await databases.createDocument(
+    'marvel',
+    'movies',
+    ID.unique(),
+    {
+        title: 'Spiderman',
+        year: 2002,
+        reviews: [
+            { author: 'Bob', text: 'Great movie!' },
+            { author: 'Alice', text: 'Loved it!' }
```

Comment:
to create the nested document, an ID must be included like:

```suggestion
            { $id: ID.unique(), author: 'Bob', text: 'Great movie!' },
            { $id: ID.unique(), author: 'Alice', text: 'Loved it!' }
```

Reply:
This was updated so that if `$id` is not included in a nested document that is being created, a unique ID will be added automatically